updated synology-cloud-station (4.0-4203)

jpalmieri opened the pull request.

jpalmieri commented:
I'm not sure why they changed the name from simply "Cloud Station" to "Cloud Station Drive" with this version but this is the same app, the OSX client for Synology Cloud Station.

jpalmieri commented:
Also, this was created using `cask-repair`, so I wasn't able to change the cask name (it gave me an error when the cask name was different from the file name). If it's as simple as making sure the file name and cask name match, let me know and I'll change them to `synology-cloud-station-drive` along with the `name` key so there's parity all around.

vitorgalvao commented:
> If it's as simple as making sure the file name and cask name match

It is. Would you rather I merge this and you submit another PR, or can you do it in this one?

jpalmieri commented:
I'll take care of it in this one. Thanks!

On Mon, Apr 4, 2016 at 11:57 AM, Vítor Galvão <notifications@github.com>
wrote:

> If it's as simple as making sure the file name and cask name match
>
> It is. Would you rather I merge this and you submit another PR, or can you
> do it in this one?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/caskroom/homebrew-cask/pull/20272#issuecomment-205446934>
>

vitorgalvao commented:
Please @mention me after you do, so I get an email to check back.

jpalmieri commented:
@vitorgalvao I decided to leave the filename and cask name as-is, since it will make upgrading this app easier.

Since you last saw this, I forced pushed a couple changes to the original commit:
* Updated the `name` key on line 6
* Updated the `pkg` path on line 10

vitorgalvao commented on Casks/synology-cloud-station.rb:
```diff
@@ -1,13 +1,13 @@
   license :gratis
 
-  pkg "synology-cloud-station-#{version.sub(%r{.*-}, '')}.pkg"
+  pkg 'Install\ Cloud\ Station\ Drive.pkg'
```
Backslashes must be removed. The string is quoted.

vitorgalvao commented:
> I decided to leave the filename and cask name as-is, since it will make upgrading this app easier.

We don’t support upgrade functionality.

Correctness is more important. Please change it.

adidalal commented:
Thank you for the contribution. It needed some fixes, so they were made in 6822111cb7d837ce83526e90ab445d470041f2da. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

jpalmieri commented:
Thanks, @adidalal. I didn't see any difference with my changes and https://github.com/caskroom/homebrew-cask/commit/6822111cb7d837ce83526e90ab445d470041f2da. Just so I do it right next time, what did I miss?

adidalal commented:
Squashed commits

jpalmieri commented:
@adidalal :+1: thanks
